package.json

```diff
@@ -36,6 +36,7 @@
     "start-clock": "node --max_old_space_size=1500 --nouse-idle-notification clock/index.js",
     "start-worker": "node --max_old_space_size=1500 --nouse-idle-notification worker/jobProcessor.js",
     "test-api": "mocha -R dot --recursive tests/api",
+    "test-api-log": "ENABLE_API_LOGS=true mocha -R dot --recursive tests/api",
```

Comment:
testing the API logger by running the API tests with the config variable on. If any tests fail here while passing without  the API logger, we know there's something wrong with the logger implementation.